ee/hogai/eval/eval_memory.py

```diff
@@ -0,0 +1,161 @@
+from typing import Optional
+import pytest
+from braintrust import Eval, EvalCase, Score
+from autoevals.llm import LLMClassifier
+from .conftest import BRAINTRUST_PROJECT_NAME
+from .scorers import ToolRelevance
+from ee.hogai.utils.types import AssistantState, AssistantNodeName
+from ee.hogai.graph import AssistantGraph
+from ee.models.assistant import Conversation
+from posthog.schema import HumanMessage, AssistantMessage, AssistantToolCall
+
+
+class MemoryContentRelevance(LLMClassifier):
```

Comment:
Does this run the evaluation against the tool call arguments?

Reply:
Yes, exactly, what's nice here is that you can actually see all the inputs/outputs/expecteds _and_ the reasoning that led to the score in the eval case's trace, [right here](https://www.braintrust.dev/app/PostHog/p/Max%20AI/experiments/memory-wip-f72ca346?c=root-wip-9545c277&r=f58e0fd7-6bf4-4f08-8ecc-30d7f1fedce1&s=08b34583-c2bd-4808-8c3f-ab8abc2824f4)